implemented gemma_3_VLM

gun29may opened the pull request:
Added support for directly using gemma3 vlm released by google using hugging face

johnnynunez commented:
> Added support for directly using gemma3 vlm released by google using hugging face

Delete cache files and i will merge it

gun29may commented:
> > Added support for directly using gemma3 vlm released by google using hugging face
> 
> Delete cache files and i will merge it

I have deleted the pycache dir

gun29may commented:
> > > > Added support for directly using gemma3 vlm released by google using hugging face
> > > 
> > > 
> > > Delete cache files and i will merge it
> > 
> > 
> > I have deleted the pycache dir
> 
> ![IMG_6663](https://private-user-images.githubusercontent.com/22727137/455149612-3fe60992-38ec-4bf8-b003-1b5f11a9d49f.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDk5MDMwNzYsIm5iZiI6MTc0OTkwMjc3NiwicGF0aCI6Ii8yMjcyNzEzNy80NTUxNDk2MTItM2ZlNjA5OTItMzhlYy00YmY4LWIwMDMtMWI1ZjExYTlkNDlmLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYxNFQxMjA2MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZTFlNjg3YWQwNDhiNzE0Y2ViNTZmMDEyMjMwMWQ3ZTlhNTViNzVkNTYxYzFhMWYwMGYwNGQ2OGY3NTk2NTI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ZtYYMWwAr8EnnKqr-Gck1pUC4SZ17uQMcOLut3bklEM)

deleted these cache directories also

johnnynunez commented:
> > > > > Added support for directly using gemma3 vlm released by google using hugging face
> > > > 
> > > > 
> > > > Delete cache files and i will merge it
> > > 
> > > 
> > > I have deleted the pycache dir
> > 
> > 
> > ![IMG_6663](https://private-user-images.githubusercontent.com/22727137/455149612-3fe60992-38ec-4bf8-b003-1b5f11a9d49f.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDk5MDMwNzYsIm5iZiI6MTc0OTkwMjc3NiwicGF0aCI6Ii8yMjcyNzEzNy80NTUxNDk2MTItM2ZlNjA5OTItMzhlYy00YmY4LWIwMDMtMWI1ZjExYTlkNDlmLmpwZWc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNjE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDYxNFQxMjA2MTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZTFlNjg3YWQwNDhiNzE0Y2ViNTZmMDEyMjMwMWQ3ZTlhNTViNzVkNTYxYzFhMWYwMGYwNGQ2OGY3NTk2NTI1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ZtYYMWwAr8EnnKqr-Gck1pUC4SZ17uQMcOLut3bklEM)
> 
> deleted these cache directories also

Thanks!
